docs/FunctionalTestBuildScripts.md

```diff
@@ -204,3 +204,7 @@ The Visual Studio 2022 "empty console application" template, compiled as Release
 ## Native_x64_VS2022_PDBPageSize_8192.exe
 
 The Visual Studio 2022 "empty console application" template, compiled as Debug|x64.  The `/PDBPageSize:8192` linker option set page size to 8192.
+
+## Sha256SignedUntrustedRoot.exe
```

Comment:
>Sha256SignedUntrustedRoot [](http://example.com/codeflow?start=3&length=25)

note that the test file added is not the same error as that user.
the included MS bulletin mentioned it is lack of security patch, so it does not look like a easy reproduce by ourselves. 

TRUST_E_MALFORMED_SIGNATURE = 0x80096011, // The digital signature of the object is malformed. For technical detail, see security bulletin MS13-098.